src/pyAEATsii/service.py

```diff
@@ -72,20 +71,17 @@ def submit(self, headers, invoices, mapper=None):
         _logger.debug(response_)
         return response_
 
-    def cancel(self, headers, invoices, mapper=None):
-        body = (
-            [mapper.build_delete_request(i) for i in invoices]
-            if mapper
-            else invoices
-        )
+    def cancel(self, headers, body, mapper=None):
```

Comment:
Please, if you are further evolving your fork, make the pull requests from different branches other than master so that features can be discussed and integrated separately.

Adittionally, I don't see the point of this particular change. Could you please explain its purpose?